plugins/volumeFilter.go

```diff
@@ -103,8 +106,25 @@ var _ SubmitFilter = &volumeFilter{}
 
 // Validate ensures validity
 func (c *VolumeFilterConfig) Validate() error {
-	if c.isEmpty() {
-		return fmt.Errorf("the volumeFilterConfig was empty")
+	if c.SellBaseAssetCapInBaseUnits != nil && c.SellBaseAssetCapInQuoteUnits != nil {
```

Comment:
drop the `Sell` prefix, like we discussed in slack

Reply:
Yup - those changes have all been made with the buy-side implementation. I'll merge that in and then update this branch.